riotfile.py

```diff
@@ -429,6 +429,18 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "msgpack": ["~=1.0.0", latest],
             },
         ),
+        Venv(
+            name="wait",
+            command="python tests/wait-for-services.py {cmdargs}",
+            pys="3.9",
```

Comment:
Would this still work with
```suggestion
            pys="3",
```
?

Reply:
Unfortunately by default `pys="3"` leads to using Python 3.10, and there's some incompatibilities with Python 3.10 and `kombu` and `vertica`, specifically related to removals/deprecations in `collections`. For example, I saw this error with kombu: `ImportError: cannot import name 'Iterable' from 'collections' (/root/.pyenv/versions/3.10.3/lib/python3.10/collections/__init__.py)`

For simplicity of this PR I just decided to pin the python for this job for a non-breaking version, especially since this job isn't high priority enough to be running on latest anyway.